cmd/template/canary.go

```diff
@@ -69,11 +69,33 @@ func canary(cmd *cobra.Command, options *templateCanaryOptions, args []string) e
 		"The pause behavior is the deployment waits until a manual approval is given to continue. Only set this to true if there is no duration pause behavior for this step.")...)
 	pauseUA.Content = append(pauseUA.Content, pauseUANode, pauseUAValuesNode)
 
-	stepsValuesNode.Content = append(stepsValuesNode.Content, pause, weight, pauseUA)
+	if len(options.features) == 0 {
+		options.features = append(options.features, "manual")
+	}
+	for _, feature := range options.features {
+		switch {
+		case feature == "manual":
+			stepsValuesNode.Content = append(stepsValuesNode.Content, pause, weight, pauseUA)
+		case feature == "automated":
+			// automated adds an analysis definition at the root level
+			analysisNode, analysisValuesNode := util.BuildMapNode("analysis", "Define queries and thresholds used for automated analysis")
```

Comment:
```suggestion
			analysisNode, analysisValuesNode := util.BuildMapNode("analysis", "Specify queries and thresholds used for automated analysis.")
```

The queries are actually defined in the analysis block, and they're picking them here, right?

At least in my head, defining == writing them out and specifying == picking

Reply:
This is actually the analysis block where they are defining the queries.